src/frequenz/sdk/timeseries/_resampling.py

```diff
@@ -457,6 +449,34 @@ def remove_timeseries(self, source: Source) -> bool:
             return False
         return True
 
+    def _sync_timer(self, extra_period: bool = True) -> datetime:
```

Comment:
Silly, but I would either remove this extra period, as is it only a startup glitch we are trying to fix (I guess it made sense in the original code because it was *free*), or to switch the default, as the normal situation seems to be not adding an extra period.